geoportailv3/static/js/maincontroller.js

```diff
@@ -211,6 +219,30 @@ app.MainController = function(
       $('app-themeswitcher #themes-content').collapse('hide');
     }
   }, this));
+
+  var selectInteraction = new ol.interaction.Select({
+    filter: goog.bind(function(feature, layer) {
+      return this.drawnFeatures.getArray().indexOf(feature) != -1;
+    }, this)
```

Comment:
As an optimization you can also add a layer filter:

```js
    layers: [ngeoFeatureOverlayMgr.getLayer()]
```

With this OpenLayers' hit detection mechanism will skip other layers.